6-media-queries/css/styles.css

```
@@ -0,0 +1,74 @@
+html{
+    background: #DCDCDC;
```

```suggestion
    background: #dcdcdc;
```
use lower case for the colors